crates/oxc_parser/src/lexer/mod.rs

```diff
@@ -365,6 +367,13 @@ impl<'a> Lexer<'a> {
         // > the rough order of frequency for different token kinds is as follows:
         // identifiers/keywords, ‘.’, ‘=’, strings, decimal numbers, ‘:’, ‘+’, hex/octal numbers, and then everything else
         match c {
+            #[cfg(target_arch = "wasm32")]
+            ' ' | '\t' => Kind::WhiteSpace,
+            #[cfg(target_arch = "wasm32")]
+            '\r' | '\n' => {
```

Comment:
We are probably missing all the irregular white spaces, you can import `is_irregular_whitespac` from crates/oxc_syntax/src/identifier.rs

Reply:
They were already running here https://github.com/Boshen/oxc/pull/436/files#diff-5b6f96408a56545d0f646674e9e1996cbc2b7eea8e26472adb718e41673f02e9R450-R454